src/java/org/apache/nutch/segment/SegmentReader.java

```diff
@@ -70,32 +69,31 @@
 import org.apache.nutch.util.HadoopFSUtil;
 import org.apache.nutch.util.NutchConfiguration;
 import org.apache.nutch.util.NutchJob;
+import org.apache.nutch.util.SegmentReaderUtil;
 
 /** Dump the content of a segment. */
-public class SegmentReader extends Configured implements Tool,
-    Reducer<Text, NutchWritable, Text, Text> {
+public class SegmentReader extends Configured implements Tool {
 
   private static final Logger LOG = LoggerFactory
       .getLogger(MethodHandles.lookup().lookupClass());
 
-  long recNo = 0L;
+  static long recNo = 0L;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.